src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -325,6 +325,245 @@ private AdsDataProvider() {
   }
 
   @DataProvider
+  public static Object[][] dfpParamsOasis() {
+    return new Object[][]{
+        {
+            "yugioh",
+            "Dark_Magician",
+            "wka.ent/_yugioh//article",
+            "TOP_LEADERBOARD",
+            Arrays.asList(
+                "\"s0\":\"ent\"",
+                "\"s0v\":\"comics\"",
+                "\"s0c\":[\"gaming\",\"anime\",\"videogames\"]",
+                "\"s1\":\"_yugioh\"",
+                "\"s2\":\"article\"",
+                "\"dmn\":\"wikiacom\"",
+                "\"hostpre\":",
+                "\"cat\":[",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\"cat\":[" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)